sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/FetchResult.cs

```diff
@@ -6,7 +6,11 @@ namespace Azure.Iot.ModelsRepository.Fetchers
     internal class FetchResult
     {
         public string Definition { get; set; }
+
         public string Path { get; set; }
-        public bool FromExpanded => Path.EndsWith("expanded.json", System.StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Former string differs from new constant in that new constant is prefixed with a period. Is that intended?

Reply:
yes, I tested this and it should be expected to have a period